app/workers/hit_fox_feed_worker.rb

```diff
@@ -0,0 +1,14 @@
+require 'hit_fox_feed'
+
+class HitFoxFeedWorker
+  include Sidekiq::Worker
+  include Sidetiq::Schedulable
+
+  recurrence { minutely.second_of_minute(0, 5, 10, 15, 20, 25) }
```

Comment:
did you check that this actually works every 5 seconds and not for the first 25 second of the minute?